src/Compilers/CSharp/Portable/BoundTree/BoundExpression.cs

```diff
@@ -286,6 +286,14 @@ internal partial class BoundBinaryOperator
         }
     }
 
+    internal partial class BoundInterpolatedString
+    {
+        public override ConstantValue? ConstantValue
```

Comment:
What actually uses this property?

Reply:
Ah, nevermind.

---
In reply to: [447344161](https://github.com/dotnet/roslyn/pull/45396#discussion_r447344161) [](ancestors = 447344161)